src/utils.rs

```diff
@@ -20,7 +18,6 @@ mod unsafe_code {
     /// Wrapper around a slice that was allocated in C code.
     ///
     /// `CSlice` is only available when the `allow-unsafe-code` feature is enabled.
-    #[derive(Debug)]
```

Comment:
Hm?

Reply:
Well, okay, I guess the `Debug` implementation for `NonNull` is not really useful, so this is actually a good idea, I guess...

Reply:
Oh, ups. A hand written implementation is added below. I should start using GitHub PR reviews...

Reply:
Exactly, the derived implementation was just printing the pointer address.